cleverpush/src/main/java/com/cleverpush/stories/StoryView.java

```diff
@@ -49,18 +64,26 @@ public StoryView(Context context, AttributeSet attributeSet) {
   }
 
   private void loadStory() {
-    if (loading) {
-      return;
+    if (attrArray.getString(R.styleable.StoryView_widget_id) == null ||
```

Comment:
Save `attrArray.getString(R.styleable.StoryView_widget_id)` into a variable before to keep code shorter